src/pudl/constants.py

```diff
@@ -6,35 +6,19 @@
 data cleaning purposes.
 """
 
-import pandas as pd
-import sqlalchemy as sa
+from typing import Any, Dict, List, Tuple
 
-######################################################################
-# Constants used within the init.py module.
-######################################################################
-prime_movers = [
-    'steam_turbine',
-    'gas_turbine',
-    'hydro',
-    'internal_combustion',
-    'solar_pv',
-    'wind_turbine'
-]
-"""list: A list of the types of prime movers"""
+import pandas as pd
 
-rto_iso = {
-    'CAISO': 'California ISO',
-    'ERCOT': 'Electric Reliability Council of Texas',
-    'MISO': 'Midcontinent ISO',
-    'ISO-NE': 'ISO New England',
-    'NYISO': 'New York ISO',
-    'PJM': 'PJM Interconnection',
-    'SPP': 'Southwest Power Pool'
-}
-"""dict: A dictionary containing ISO/RTO abbreviations (keys) and names (values)
-"""
+from pudl.metadata.fields import (CUSTOMER_CLASSES, EPACEMS_MEASUREMENT_CODES,
+                                  FUEL_CLASSES, NERC_REGIONS,
+                                  RELIABILITY_STANDARDS, REVENUE_CLASSES,
+                                  TECH_CLASSES)
+from pudl.metadata.labels import (ENTITY_TYPES, ESTIMATED_OR_ACTUAL,
+                                  FUEL_TRANSPORTATION_MODES_EIA,
+                                  MOMENTARY_INTERRUPTIONS)
 
-us_states = {
+US_STATES_TERRITORIES: Dict[str, str] = {
```

Comment:
I am all for killing `pudl.constants` (See our oldest surviving issue: #52). I didn't pull apart the states and territories because we need to think a little bit about how to combine these different categories well.